frontend/src/pages/Results/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { Category, Choice, Question } from '../../actions';
+import { useEffect, useState } from 'react';
+import { config } from '../../actions/config';
+import { SessionData, UserProgress, Result } from '../../actions/types';
+import axios from 'axios';
+import { connect } from 'react-redux';
+
+interface Props {
+  category: Category;
+  questions: Question[];
+  choices: Choice[][];
+  currentLogin: SessionData;
+}
+
+const ResultsPage = ({
+  currentLogin: { user, token },
+  category,
+  questions,
+  choices,
+}: Props) => {
+  const [score, setScore] = useState(0);
+  const [quizData, setQuizData] = useState<Result[]>();
+
+  useEffect(() => {
+    getUsersQuizData();
+  }, []);
+
+  useEffect(() => {
+    checkQuizStatus().then((res) => {
+      res.status === -1 && updateStatus(res.id);
```

Comment:
PLease create a constant or enum on `-1` because we cannot not know what kind of status is this